assets/src/styles/blocks/Gallery_carousel.scss

```diff
@@ -57,6 +57,7 @@
     img {
       width: 100%;
       object-fit: cover;
+      cursor: pointer;
```

Comment:
I guess we also need to add `cursor: pointer` for `.wp-block-image:not(force-no-lightbox)`?